docs/tutorial_docker.md

```diff
@@ -1,10 +1,100 @@
+   ```
+
+   - `/path/to/your/data`: The path on your host machine where your data is located.
+   - `/mnt/data`: The directory inside the container where the host directory will be mounted.
+   
+   Now, any files placed in `/path/to/your/data` will be accessible from `/mnt/data` inside the Docker container. This allows you to read input files and save output directly from/to your local machine while running FragPipe in Docker.
+
+3. **Running FragPipe Inside the Docker Container:**
+
+   To run FragPipe inside the container, mount the required directories (for example, where your data or project files are located), then execute the FragPipe tool. You can use the following command:
+
+   ```bash
+   docker run -v /path/to/your/data:/mnt/data -it fcyucn/fragpipe:<version> /fragpipe_bin/FragPipe-<version>/fragpipe/bin/fragpipe -config /mnt/data/config_file.fpconfig
+   ```
+
+   - `-v /path/to/your/data:/mnt/data`: Mounts the host directory to the container.
+   - `-it`: Enables interactive mode, allowing you to interact with the container.
+   - `/fragpipe_bin/FragPipe-<version>/fragpipe/bin/fragpipe`: Executes FragPipe inside the container.
+   - `-config /mnt/data/config_file.fpconfig`: Specifies the configuration file from the mounted directory for FragPipe.
+
+   You can replace `/mnt/data/config_file.fpconfig` with the path to your actual configuration file located in the mounted directory. This way, FragPipe runs with your specific settings.
+
+4. **Save the Docker image to a TAR file:**
+
+   To convert the Docker container for use in Singularity, save the Docker image to a tarball:
+
+   ```bash
+   docker save -o fragPipe-<version>.tar fcyucn/fragpipe:<version>
+   ```
+
+   This command will export the Docker image into a file called `fragPipe-<version>.tar`.
+
+## Running FragPipe with Singularity
+
+1. **Convert the Docker image to a Singularity image:**
+
+   Use the exported Docker tarball to create a Singularity image:
+
+   ```bash
+   singularity build fragPipe-<version>.img docker-archive://fragPipe-<version>.tar
+   ```
+
+   This will convert the Docker image into a Singularity image named `fragPipe-<version>.img`.
+
+2. **Run the FragPipe tool with Singularity:**
+
+   Once the Singularity image is built, you can execute the `fragpipe` command with:
+
+   ```bash
+   singularity exec fragPipe-<version>.img /fragpipe_bin/FragPipe-<version>/fragpipe/bin/fragpipe --help
+   ```
+
+   This command runs the same `fragpipe` tool using the Singularity image, displaying the help options.
+
+3. **Mounting Host Directories with Singularity:**
+
+   Similar to Docker, you can mount directories from your host system when using Singularity. Use the `-B` option to bind a directory from your host system to the Singularity container:
+
+   ```bash
+   singularity exec -B /path/to/your/data:/mnt/data fragPipe-<version>.img /fragpipe_bin/FragPipe-<version>/fragpipe/bin/fragpipe -config /mnt/data/config_file.fpconfig

```

Comment:
What is the -config flag for? And what is the format of the fpconfig file? Why not just provide FragPipe's arguments?